pkg/subctl/cmd/gather/connectivity.go

```diff
@@ -0,0 +1,120 @@
+/*
+© 2021 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gather
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"github.com/submariner-io/submariner-operator/pkg/discovery/network"
+)
+
+const (
+	gatewayPods = "app=submariner-gateway"
+	routeagentPods = "app=submariner-routeagent"
+)
+
+func GetGatewayPodsLogs(clientSet kubernetes.Interface, clustername string) error {
```

Comment:
Same in other functions as well. In general, please follow the camel-case naming convention when a var has multiple words.  

```suggestion
func GetGatewayPodsLogs(clientSet kubernetes.Interface, clusterName string) error {
```

Reply:
Re: the function name, the package name is `gather` so we can just name the function `GatewayPodLogs` as it will be referenced as `gather.GatewayPodLogs`. Same with the other functions